### DISCUSSION TITLE
Using Imgur for sprites

### DISCUSSION THREAD

**Discussion by MansiSeth** [replit/kaboom#670] [Q&A]
I am new to JS and Kaboom and i was trying to pull sprites hosted on imagur but they aren't getting rendered

<img width="959" alt="image" src="https://user-images.githubusercontent.com/81612581/214518734-a6c245bd-060c-4ec0-ab5a-c627f8a8286c.png">

**Comment by lajbel** [comment]
You are using a very old version, btw.
You can show your code? Sorry for late response
